src/symbol_table/SymbolTable.ts

```diff
@@ -2,6 +2,7 @@ import {Action, Flow} from "../ast";
 import Value from "../ast/Value";
 import FlowSymbolTable from "./FlowSymbolTable";
```

Comment:
@ginabolo Try removing this import. I think this is where the circular dependency is coming in.

Reply:
Yes but don't I need that so that the flows can map to FlowSymbolTables? The SymbolTable imports FlowSymbolTable, and FlowSymbolTable obviously imports the SymbolTable because it extends it.

Reply:
Okay I moved both definitions to the same file ¯\_(ツ)_/¯
Working on passing the tests.

Reply:
This is fine for now, but not best practice.
In general, you shouldn't need to import FlowSymbolTable in SymbolTable.
This problem is typically solved through interface segregation, or using generic types: https://www.typescriptlang.org/docs/handbook/generics.html

Reply:
( But don't worry about fixing it for now - our top priority is getting a working parser out the door ;) )